pkg/cmd/repo/autolink/view/view.go

```diff
@@ -0,0 +1,103 @@
+package view
+
+import (
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/internal/browser"
+	"github.com/cli/cli/v2/internal/ghrepo"
+	"github.com/cli/cli/v2/pkg/cmd/repo/autolink/shared"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type viewOptions struct {
+	BaseRepo       func() (ghrepo.Interface, error)
+	Browser        browser.Browser
+	AutolinkClient AutolinkViewClient
+	IO             *iostreams.IOStreams
+	Exporter       cmdutil.Exporter
+
+	ID string
+}
+
+type AutolinkViewClient interface {
+	View(repo ghrepo.Interface, id string) (*shared.Autolink, error)
+}
+
+func NewCmdView(f *cmdutil.Factory, runF func(*viewOptions) error) *cobra.Command {
+	opts := &viewOptions{
+		Browser: f.Browser,
+		IO:      f.IOStreams,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "view <id>",
+		Short: "View an autolink reference",
+		Long:  "View an autolink reference for a repository.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.BaseRepo = f.BaseRepo
+			httpClient, err := f.HttpClient()
+
+			if err != nil {
+				return err
+			}
+
+			opts.ID = args[0]
+
+			if err != nil {
+				return err
+			}
+
+			opts.AutolinkClient = &AutolinkViewer{HTTPClient: httpClient}
```

Comment:
Do we need the extra `err` check?  Is this functionally the same but easier to follow?

```suggestion
		RunE: func(cmd *cobra.Command, args []string) error {
			httpClient, err := f.HttpClient()
			if err != nil {
				return err
			}

			opts.BaseRepo = f.BaseRepo
			opts.ID = args[0]
			opts.AutolinkClient = &AutolinkViewer{HTTPClient: httpClient}
```

Reply:
This was purely an oversight on my part. The second error check is superfluous. Thanks!